pkgs/tools/security/sudolikeaboss/deps.nix

```diff
@@ -0,0 +1,22 @@
+# This file was generated by go2nix: https://github.com/kamilchm/go2nix
+# v1.1.0 or 1.1.1, not 100% sure
```

Comment:
I would love that feature! It can be so helpful, when coming back to update some code, to know the version it was generated on.